sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -133,9 +133,7 @@ public Integer getCount() {
     }
 
     @Override
-    public List<TaskSpec> getTasks() {
-        return tasks;
-    }
+    public List<TaskSpec> getTasks() { return tasks; }
```

Comment:
Nit: let the computer fold blocks, keeping a full indented block around saves keystrokes when you need to make a quick edit, and IMO I think it also has the readability advantage.